components/DisplayRC.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import Image from "next/image";
+
+// helper function used in the Image component
+function cn(...classes) {
+  return classes.filter(Boolean).join(" ");
+}

```

Comment:
This looks like a useful utility that will be required across the project. We should move it to the `util` folder instead to share it across the components. _(the folder structure needs some love, but that's another issue altogether)_

The comment doesn't add a lot of context, maybe we do not need it.

Reply:
I moved the function to the ``` util ``` folder and named it ```booleanFilter``` to better represent what the function does. I also added a comment that gives it better context.